dev-resources/puppetlabs/services/master/tasks_int_test/echo

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+echo $@
```

Comment:
Also note to myself, need to fix these up with a version that returns the correct file content (now that we're checking the hash). Integration test fails currently.